mftf/2.3/getting-started.md

```diff
@@ -67,26 +69,27 @@ git clone git@github.com:magento/magento2.git
 
 ### Step 2. Install dependencies {#install-dependencies}
 
-1. Checkout to the Magento version that you are going to test.
+Checkout to the Magento version that you are going to test.
```

Comment:
I would remove "to":   Checkout the Magento version that you are going to test.

Reply:
:ok: